components/homme/src/tool/CMakeLists.txt

```diff
@@ -0,0 +1,74 @@
+# macro for building the tools executable
+
+macro(tool_setup)
+
+SET(TARGET_DIR        ${HOMME_SOURCE_DIR}/src/tool)
+SET(UTILS_SHARE_DIR   ${HOMME_SOURCE_DIR}/utils/csm_share)
+SET(SRC_DIR           ${HOMME_SOURCE_DIR}/src)
+SET(SRC_SHARE_DIR     ${HOMME_SOURCE_DIR}/src/share)
+SET(TEST_SRC_DIR      ${HOMME_SOURCE_DIR}/src/test_src)
```

Comment:
Probably tools won't need test_src?

Reply:
Test modules get used in various spots in the code, so it's needed to build.

Reply:
this is very minor: I am only asking because there is a code in tests_src that needs commenting if one wants to run intel with checks, for example. I thought tools won't need tests, so, it would make debugging easier. or maybe build tools with CAM=true directive.

Reply:
You might open an issue about "code in tests_src that needs commenting if one wants to run intel with checks," as that implies something is broken.

What I meant above is that there is core code that uses `test_mod`, which in turn uses modules in `test_src`. So new cpp code would be needed to isolate `test_src` sufficiently. I consider that orthogonal to this PR. I believe setting CAM to true would create unresolved `use`s of modules from CAM.